pallets/did/src/lib.rs

```diff
@@ -848,9 +928,9 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
-	// Check if the provided block number is valid,
-	// i.e., if the current blockchain block is in the inclusive range
-	// [operation_block_number, operation_block_number + MaxBlocksTxValidity].
+	/// Check if the provided block number is valid,
```

Comment:
This is not a public function, why are we making RustDocs for it?

Reply:
Why do we put a comment in front if it's private? If it's a doc comment you see the comment at the call site when you hover over the function. Private or not this is a useful feature.

Reply:
> Why do we put a comment in front if it's private? If it's a doc comment you see the comment at the call site when you hover over the function. Private or not this is a useful feature.

Exactly this!

Reply:
Aren't they then also exported as RustDoc?

Reply:
Rust is smart. Rust doesn't show private functions in it's doc. Be like Rust!